.devcontainer/devcontainer.json

```diff
@@ -29,7 +29,7 @@
 	"forwardPorts": [45883],
 
 	// Use 'postCreateCommand' to run commands after the container is created.
-	// "postCreateCommand": "go version",
+	"postCreateCommand": "make depscheck",
```

Comment:
I'm wondering what does this command do?

Reply:
This command is executed when the Remote Container is started. This command is not relevant to this pull request, so it is reverted. (I am now considering whether to include it in `Dockerfile` or `devcontainer.json`). I will propose this topic in a another pull request.

Reply:
I removed the `postCreateCommand` comment generated by VS code. Because `postCreateCommand` is not used. I will send a new pull request if I want to use `postCreateCommand`.

Reply:
@magodo Thank you for your quick response and for picking it up. Please review again. 🙇